Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Undo/PrefabUndoUtils.cpp

```diff
@@ -16,16 +16,6 @@ namespace AzToolsFramework
     {
         namespace PrefabUndoUtils
         {
-            void AppendAddEntityPatch(
-                PrefabDom& patches,
-                const PrefabDomValue& newEntityDom,
-                const AZStd::string& newEntityAliasPath)
-            {
-                AZ_Assert(patches.IsArray(), "AppendAddEntityPatch - Provided patches should be an array object DOM value.");
-
-                AppendUpdateValuePatch(patches, newEntityDom, newEntityAliasPath, PatchType::Add);
-            }
```

Comment:
Having clear named functions like this can actually help readability. In fact,  I'd recommend having functions like these call the underlying 'AppendUpdateValuePatch' and 'AppendUpdateValuePatchByComparison' functions rather than trying to route all calls to those functions.